content/admin/organization/insights.md

```diff
@@ -0,0 +1,188 @@
+| Users opted out of analytics | The number of users that are a member of your organization that have opted out of sending analytics. <br><br>When users opt out of sending analytics, you won't see any of their data in Insights. To ensure that the data includes all users, you can use [Settings Management](/desktop/hardened-desktop/settings-management/) to set `analyticsEnabled` for all your users.                                                                                                                                                                                                                                                                                                                                                                                          |
+| Active users (graph)         | The view over time for total active users.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+
+
+### Builds
+
+Monitor development efficiency and the time your team invests in builds with
+this chart. It provides a clear view of the build activity, helping you identify
+patterns, optimize build times, and enhance overall development productivity.
+
+The chart contains the following data.
+
+| Data                   | Description                                                                                                                                                                                        |
+|:-----------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Average build per user | The average number of builds per active user. A build includes any time a user runs `docker build`, `compose build`, `compose up --build`, or any command beginning with `docker buildx`.          |
```

Comment:
This isn't right, flagging all `buildx` invocations as a build . `buildx` has many other commands for building. We should narrow this data point down to `buildx build` and `buildx bake`, and ignore any other `buildx` commands.

If we can't change the data, then we should at least probably call this discrepancy out in this doc.

Reply:
Thank you. Got a more exact definition and updated it.